lib/dev-server.js

```diff
@@ -50,9 +52,18 @@ async function devServer(options) {
         app.use(fallback(entryPoint, { root: options.contentBase }));
     }
 
-    app.listen(options.port);
+    if (options.https) {
+        if (!(options.key && options.cert)) {
+            console.log('Usage of --https requires --cert and --key to be set.')
```

Comment:
@PepsRyuu do you have a good idea how to exit here? I tried throwing an error and process.exit but both weren't really nice from a user perspective (alwas stack trace and the log statement hidden in there).

Reply:
Considering the check is inside ```dev-server```, not sure it would be a good idea to cause the process to exit forcefully. Sounds like that could have unintended consequences. I think it would be better to ```throw new Error('...')```.

Reply:
I just pushed a commit switching to an error here and capturing it inside the cli to be a little bit more human friendly. What do you think?

Reply:
I like it! Works well! Happy to merge whenever you think it's ready.

Reply:
LGTM :)